enterprise/backend/src/metabase_enterprise/remote_sync/task/import.clj

```diff
@@ -31,10 +32,12 @@
         (let [{task-id :id existing? :existing?} (impl/create-task-with-lock! "import")]
           (if existing?
             (log/info "Remote sync already in progress, not auto-importing")
-            (dh/with-timeout {:interrupt? true
-                              :timeout-ms (* (settings/remote-sync-task-time-limit-ms) 10)}
-              (log/info "Auto-importing remote-sync collections")
-              (impl/handle-task-result! (impl/import! source task-id) task-id))))))))
+            (task-history/with-task-history {:task "remote-sync-auto-import"
```

Comment:
i wonder whether user triggered sync should also be tracked with task history. One example is sending notification via api still have task history tracked.

Reply:
hmmm good point. I think manual db/table syncs also get tracked

Reply:
actually, the description of system tasks is ```internal processes tasks```, so I don't think manually used syncs would qualify. i'm tepmted to leave as is. we log manual import/exports in the audit log